app/helpers/left_menu_bar_helper.rb

```diff
@@ -0,0 +1,31 @@
+module LeftMenuBarHelper
+  def projects_are_selected?
+    controller_name.in? %w(projects experiments my_modules)
+  end
+
+  def repositories_are_selected?
+    controller_name == 'repositories'
+  end
+
+  def templates_are_selected?
+    # TBD
```

Comment:
I know I'm veery much nitpicking, but I'd change these mentions here to `#TODO` because it's a more common denominator for "work remaining to be done", and there's a much higher chance it'll pop up in some search for "TODOs".